### PR TITLE
[Bugfix]: Judge Views

### PR DESCRIPTION
**Pull request by mkanoria**
**Bugfix**
- Hide Add Award button for judge view

**Feature**
- Add delete award button for score based awards
- Add audio player component to awards details

Should have broken this into multiple PR's but they're all small features so figured I'd get them in together. Can make separate PR's if needed

### CODE REVIEW

File: components/awards/ScoreBasedAwards.js [hunk 1]
```diff
@@ -113,6 +117,23 @@ export default function Performances({ award }) {
     }
   }
 
+  async function deleteAward() {
+    setLoading(true);
+    try {
+      await axios({
+        method: 'PUT',
```
Comment (thepenguinco) [inline components/awards/ScoreBasedAwards.js:124]: POST?

**Review by thepenguinco — APPROVED** [review]
LGTM.

File: components/awards/ScoreBasedAwards.js (hunk 1 above)
Comment (mkanoria) [reply components/awards/ScoreBasedAwards.js:124]: I think we call `PUT` everywhere else, but I agree, it should be `POST`

**Comment by mkanoria** [comment]
Need to do some more testing of the Audio Player change locally, will merge after if everything looks good

File: pages/awards/[id].js [hunk 2]
```diff
@@ -367,12 +367,16 @@ const IndividualFeedback = ({ feedback }) => {
...
+      <div className={styles.judge__feedback_audio_player_wrapper}>
+        <FeedbackAudio
+          audioUrl={feedback.audio_url}
+          edit={false}
+          setRecordingChanged={() => {}}
+          recording={null}
+          setRecording={() => {}}
+          recordedBlob={null}
+          setRecordedBlob={() => {}}
+        />
```
Comment (OustanDing) [inline pages/awards/[id].js:379]: If we're only allowing playback here, please use the `AudioPlayer` component.

Comment (mkanoria) [reply pages/awards/[id].js:379]: Good catch, will update!

**Pull request merged**